libs/sr-stats/Data/Characters/Arlan/data.json

```diff
@@ -400,5 +400,44 @@
       "crit_dmg_": 0.5000000004656613,
       "taunt": 125
     }
-  ]
+  ],
+  "rankMap": {
+    "1": {
+      "skillTypeAddLevel": {},
+      "params": [
+        0.10000000009313226
```

Comment:
values here looks like they should be `extrapolateFloat` to `0.1`

Reply:
I get a ton of these errors with the new extrapolateFloat calls, and none of these loooong numbers have been fixed
<img width="407" alt="image" src="https://github.com/frzyc/genshin-optimizer/assets/36019388/b62b33fc-908e-4845-beea-4d9dd9712fe5">

only a very small subset are fixed 
<img width="153" alt="image" src="https://github.com/frzyc/genshin-optimizer/assets/36019388/74e9e1be-10d0-47de-919c-43b8a91fe3cd">